index.php

```diff
@@ -1,11 +1,32 @@
 <?php
 
+use controllers\GraphQLController;
+
 require './vendor/autoload.php';
 
 $container = require('./config/container.php');
 
 $app = new \Slim\App($container);
 
-$app->post('/graphql', 'GraphQLController:process');
+/**
+ * Enable CORS for all routes
+ * Please change the `origin` header according to requirement
+ */
+$app->options('/{routes:.+}', function ($request, $response, $args) {
+    return $response;
+});
+
+$app->add(function ($req, $res, $next) {
+    $response = $next($req, $res);
+    return $response
```

Comment:
Here is the problem. You should not return the `response`, but call the `next` callback and return it.

```
$response = $response->withHeader('Access-Control-Allow-Origin', '*')
  ->withHeader('Access-Control-Allow-Headers', 'X-Requested-With, Content-Type, Accept, Origin, Authorization')
  ->withHeader('Access-Control-Allow-Methods', 'GET, POST, PUT, DELETE, PATCH, OPTIONS');

return $next($request, $response);
```

If you will do it like this, you can remove `$app->options('/{routes:.+}', function ...)`